src/pages/Admin/Details/ValidationSchema.ts

```diff
@@ -151,16 +142,7 @@ export const roomCreateSchema = Yup.object().shape<RoomSchemaShape>({
   venueName: Yup.string()
     .when("useUrl", {
       is: false,
-      then: Yup.string()
-        .required("Venue name is required")
-        .min(
-          VENUE_NAME_MIN_CHAR_COUNT,
-          ({ min }) => `Name must be at least ${min} characters`
-        )
-        .max(
-          VENUE_NAME_MAX_CHAR_COUNT,
-          ({ max }) => `Name must be less than ${max} characters`
-        ),
+      then: roomTitleSchema,
```

Comment:
This used to say "Venue name is required", it seems that `roomTitleSchema` says "Room name is required"

Is it still valid now? Do both of these actually refer to the same concept? If so, is 'venue' or 'room' more correct here? If not, can we make the wording more generic and just use 'Name is required' or similar?

Reply:
I was wondering what I should do about this. I'll change it to 'Name is required'

Reply:
Sounds good.